build/pkgs/gap/spkg-configure.m4

```diff
@@ -2,7 +2,7 @@ SAGE_SPKG_CONFIGURE([gap], [
   # Default to installing the SPKG, if the check is run at all.
   sage_spkg_install_gap=yes
 
-  m4_pushdef([GAP_MINVER],["4.12.2"])
+  m4_pushdef([GAP_MINVER],["4.13.0"])
```

Comment:
Just to mention it, I released GAP 4.13.1 last week. There are no build system changes and also overall I'd hope there are no changes problematic for SageMath. But of course you may wish to focus on getting 4.13.0 done first.

Reply:
Thanks, I've updated it